Readme.md

```diff
@@ -3,24 +3,28 @@
 Tool to add stock monitoring workflows to an existing [CHT](https://github.com/medic/cht-core) application.
 
 # Installation
+Clone the project in your directory, then install the package using npm:
+
```

Comment:
Is it installing the package or cloning ?
What project is to be cloned here  if it is another project then I would suggest this sentence: `Clone the project in which you would like to add the package and then install the package using npm`.